examples/albert/run_trainer.py

```diff
@@ -58,36 +58,6 @@ def get_model(training_args, config, tokenizer):
     return model
 
 
-def get_optimizer_and_scheduler(training_args, model):
```

Comment:
I've moved this to the place just before creating `hivemind.Optimizer` for clarity.